.github/workflows/ios.yml

```diff
@@ -33,6 +33,9 @@ jobs:
 
       - name: Generate
         run: |
+          # Disabling NEON fixes linker error, see
+          # https://github.com/unomed-dev/react-native-matrix-sdk/issues/12
+          export CARGO_FEATURE_NO_NEON=1
```

Comment:
This [comment](https://github.com/matrix-org/matrix-rust-sdk/issues/4416#issuecomment-2545144103) made me think that we should probably force the reldbg profile during building instead. However, uniffi-bindgen-react-native [only supports release and debug](https://github.com/jhugman/uniffi-bindgen-react-native/blob/main/crates/ubrn_common/src/rust_crate.rs#L25) right now. So I think this is fine for now.

Reply:
Have logged https://github.com/jhugman/uniffi-bindgen-react-native/issues/190 upstream to allow specifying the profile in future.